jinahub/indexers/storage/RedisStorage/README.md

```diff
@@ -0,0 +1,73 @@
+# RedisDBIndexer
+
+Key-value indexer wrapper around the Redis database. Redis is an in-memory key-value storage system. You can read more about it here: https://redis.io/
+
+**NOTE** You will need to manually delete the keys in Redis if you choose to delete a workspace. 
+
+## Snippets:
```

Comment:
Adapt README to follow the new structure

Reply:
updated